spec/github_helper_spec.rb

```diff
@@ -92,6 +92,30 @@
     end
   end
 
+  describe 'get_last_milestone' do
+    let(:test_repo) { 'repo-test/project-test' }
+    let(:last_stone) { mock_milestone('10.0') }
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        list_milestones: ['9.8 ❄️', '9.9'].map { |title| mock_milestone(title) }.append(last_stone)
+      )
+    end
+
+    before do
+      allow(described_class).to receive(:github_client).and_return(client)
+    end
+
+    it 'returns correct milestone' do
+      expect(client).to receive(:list_milestones)
+      expect(described_class.get_last_milestone(repository: test_repo)).to eq(last_stone)
+    end
```

Comment:
Thanks for adding unit tests for those!

I think it would be worth to add some more, to cover for cases like unexpected milestone names (e.g. `alpha.1`), or the `last_stone` being the frozen one with the emoji, and the likes. That can totally be for a separate PR, so that we don't block this and can get the fix for the upcoming code freeze, but still a nice addition to keep in mind.